XP-Pen_Star06/index.md

```diff
@@ -0,0 +1,19 @@
+---
+VID: "28bd"
+PID: "0078"
+vendor: XP-Pen
+product: XP-Pen Star06
```

Comment:
Please do not put vendor name into the `product` field. Please observe the model name exactly. The original name of this tablet is "Star 06", not "Star06", as seen on XP-Pen website. Please correct here, and in "tablet identifiers" used for the directory and photo file names.